pkg/command/options.go

```diff
@@ -162,6 +167,18 @@ func ParseOptions(args []string) (Options, error) {
 		}
 	}
 
+	if opts.BookmarksOnly == true {
+		if opts.URL != "" {
+			return opts, errors.New("--url not supported in bookmarks-only mode")
+		}
+		if opts.Host != "localhost" {
```

Comment:
Could you add some tests for these checks? There are some examples in `options_test.go` file that you can use as a reference.